catalyst-gateway/bin/src/service/api/cardano/cip36/filter.rs

```diff
@@ -493,29 +495,54 @@ async fn get_all_invalid_registrations(
             continue;
         };
 
+        let mut problem_report = format!("{:?}", row.problem_report);
```

Comment:
problem report should be json.
If we are going to amend it, we should convert it back into a problem report type, add the necessary extra entries then convert it back into json.
Not manually add text to it.

Reply:
Forgot to remove this, was using it to debug. Reverted.